lang/en/tool_objectfs.php

```diff
@@ -148,7 +148,7 @@
 $string['settings:openstack:container_help'] = 'The name of the container that will store the objects.';
 
 $string['settings:filetransferheader'] = 'File Transfer Settings';
-$string['settings:sizethreshold'] = 'Minimum size threshold (KB)';
+$string['settings:sizethreshold'] = 'Minimum size threshold';
```

Comment:
I think this should have the units just the correct one

Reply:
Done